src/Geocoder.php

```diff
@@ -136,26 +144,20 @@ public function isValid3wa($input)
   // Prepare the call to the API server
   private function performRequest($command, $parameters)
   {
-    // add the apikey to the parameters
-    $parameters["key"] = $this->apiKey;
```

Comment:
Might be worth documenting this as a deprecation - curious if switching to header-based authentication will improve our caching 🤔

Reply:
This is more of an internal detail on how we pass our api key to our api. I felt moving this to the header is a better approach (as to not expose the keys on the url).

Reply:
This is more of an internal detail on how we pass our api key to our api. I felt moving this to the header is a better approach (as to not expose the keys on the url).